examples/pj_init_test.c

```diff
@@ -0,0 +1,238 @@
+/*******************************************************************************
+    Tiny test of an evolving new API, demonstrating examples of init-file
+    handling.
+
+    NOTE!!!! set PROJ_LIB=. before running this
```

Comment:
Tests were tripped due to the scaling functions in pj_inv*/pj_fwd*. This is repaired for now, but a better solution is possible for "non-classic" style functions (cf comments in a recent commit).

Regarding pj_init_test.c in general: It has been removed from the repository now, and may be replaced by a pj_pipeline_test.c demo, once the final parts (molodensky transformations and tran.c) are done.